MapboxVisionAR/Render/Renderer/ARRenderer.swift

```diff
@@ -139,11 +139,13 @@ class ARRenderer: NSObject {
 
     func set(laneVisualParameters: LaneVisualParams) {
         if let arLaneNode = scene.arLaneNode() {
+            let lightPositionInRenderCoordinate = float3(Float(-laneVisualParameters.lightPosition.y),
```

Comment:
There's already a function that does this conversion in `ARRenderer`. Please extract it and reuse.

Reply:
The improvement is in #157